Goobi/newpages/ProjekteBearbeiten.jsp

```diff
@@ -223,7 +223,7 @@
 																	monthYearRowClass="monthYearRowClass"
 																	imageLocation="/newpages/images/calendarImages"
 																	popupButtonImageUrl="/newpages/images/calendarImages/calendar.gif" />
-																<%--popupButtonImageUrl="images/calender.gif"--%>
+																<%--popupButtonImageUrl="images/calendar.gif"--%>
```

Comment:
This file does not exist in the source tree, neither as `images/calender.gif` nor as `images/calendar.gif`. I could also simply remove that comment line.